src/main/java/uk/gov/pay/ledger/transaction/dao/TransactionDao.java

```diff
@@ -70,7 +70,10 @@
             "ORDER BY t.created_date DESC OFFSET :offset LIMIT :limit";
 
     private static final String SEARCH_TRANSACTIONS_CURSOR =
-            "SELECT * FROM transaction t " +
+            "SELECT t.*, po.paid_out_date AS paid_out_date FROM transaction t " +
+            "LEFT OUTER JOIN payout po on " +
+            "t.gateway_payout_id = po.gateway_payout_id " +
+            ":payoutAccountId " +
```

Comment:
Can we name \:payoutAccountId to something different (\:oayoutSearchFields or ..) to avoid confusing it as bind variable for field rather than join condition. For a moment I got confused and read it as field (as we do this for all other fields like \:gatewayAccountId, \:externalId) rather than condition